packages/e2e-tests/specs/manage-reusable-blocks.test.js

```diff
@@ -16,7 +16,7 @@ describe( 'Managing reusable blocks', () => {
 	 * @return {Promise} Promise resolving to number of post list entries.
 	 */
 	async function getNumberOfEntries() {
-		return page.evaluate( () => document.querySelectorAll( '.entry' ).length );
+		return page.evaluate( () => document.querySelectorAll( '.hentry' ).length );
```

Comment:
`.hentry`? 🙃

Reply:
Yeah I'm not really sure how this ever worked. The rows in the Manage All Blocks screen have class `hentry`. We could maybe consider selecting `.type-wp_block` instead.

<img width="840" alt="Screen Shot 2019-09-25 at 16 17 24" src="https://user-images.githubusercontent.com/612155/65574153-05706180-dfb0-11e9-8219-b1542af29efb.png">